src/main/java/se/sundsvall/smloader/integration/openemapper/lamnasynpunkt/LamnaSynpunkt.java

```diff
@@ -0,0 +1,35 @@
+package se.sundsvall.smloader.integration.openemapper.lamnasynpunkt;
+
+import se.sundsvall.smloader.integration.util.annotation.XPath;
+
+record LamnaSynpunkt(
+
```

Comment:
Why have you decided to use swedish attribute names here?
The xpath-attributes seems to be in english, why not stick to that language?

(LamnaSynpunkt should also be renamed i.m.h.o.)